include/evm2wasm.h

```diff
@@ -351,9 +351,9 @@ struct WastCode
     std::string imports;
 };
 
-std::string evm2wasm(const std::string& input);
+std::string evm2wasm(const std::vector<char>& input);
```

Comment:
Why change the public API? And what is the desired content after the change?

Reply:
This is bad choice of the type. If you want to handle multiple input types consider using `string_view` (not in standard yet) or pass iterators.